### ISSUE TITLE
UI Redesign - Mobile - Terms & Conditions

### ISSUE THREAD

**Issue by ksanty** [atsign-foundation/atmosphere_pro#220]
**Description**
The task is to implement the new design for the screens displayed on the Terms & Conditions navigation in the mobile app. 

**Assets and Design Screens** 
All assets and design screens have already been created.
- The Figma **TERMS & CONDITIONS** design can be found here under **SETTINGS** https://www.figma.com/file/gnhrfcg8OYvAz0zJXZz2r3/Atmosphere-Pro?node-id=2%3A3
- The Assets required can be found here https://www.figma.com/file/gnhrfcg8OYvAz0zJXZz2r3/Atmosphere-Pro?node-id=0%3A1

For help with using Figma please watch this short tutorial video
https://user-images.githubusercontent.com/69275762/192322842-2ac40f88-c396-4a66-933b-3c8abae516c3.mov

**Additional context**
- This work should be done on the UI_Redesign branch https://github.com/atsign-foundation/atmosphere_pro/tree/ui_redesign

**Comment by tanmoy27112000** [comment]
@ksanty I would like to work on this UI

**Comment by ruqaiyasattar** [comment]
I would love to redesign settings UI

**Comment by ksanty** [comment]
> @ksanty I would like to work on this UI

Great.  Thanks so much.  Let us know if you have questions.

**Comment by ksanty** [comment]
> I would love to redesign settings UI

Hi there, the screens posted are already redesigned and ready for implementation.  What enhancements are you proposing?  Thank you!

**Comment by tanmoy27112000** [comment]
@ksanty  in the sigma design I can see that only one screen it there the page when we navigate to using T&C button is not present. can you help me with this

**Comment by ksanty** [comment]
@tanmoy27112000 that's correct.  All the assets are under the Design Systems link as stated above - https://www.figma.com/file/gnhrfcg8OYvAz0zJXZz2r3/Atmosphere-Pro?node-id=0%3A1

**Comment by davidaodejobi** [comment]
Can you assign this to me @tinashe404? Seems there is only a person working on this... thank you
